### PR TITLE
Production deployment

### PR DESCRIPTION
**Pull request by slawosz**

### CODE REVIEW

File: .github/workflows/build-docker-prod.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: Build dev docker
```
Comment (slorek) [inline .github/workflows/build-docker-prod.yml]: Filename is `prod` but description says `dev`. I'm not sure we need to specify either? "Build and push Docker image" might be a better name.

File: .github/workflows/build-docker-prod.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" git submodule init
+        GIT_SSH_COMMAND="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no" git submodule update --recursive
+    - name: Build and push docker
+      run: |
+        export BUILD_ID=latest
```
Comment (slorek) [inline .github/workflows/build-docker-prod.yml]: We're using `master` tag in https://github.com/UKGovernmentBEIS/beis-opss-psd/blob/master/docker-compose.yml - so we need to change this?

File: .github/workflows/production-deploy.yml [hunk 3]
```diff
@@ -1,49 +1,50 @@
...
-#         cf7 auth
-#         cf7 target -o 'beis-opss' -s $SPACE
-#         chmod +x ./deploy.sh
-#         ./deploy.sh
-#         cf7 logout
-#     - name: Build Docker
-#         docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-#         # docker build
-#         # docker tag
+name: Deploy production
```
Comment (slorek) [inline .github/workflows/production-deploy.yml]: *staging and production

File: .github/workflows/build-docker-prod.yml (hunk 2 above)
Comment (slawosz) [reply .github/workflows/build-docker-prod.yml]: Yes, as discussed

**Pull request merged**